### PR TITLE
Add blog post about 1.3.0 release

### PR DESCRIPTION
**Pull request by flavio**
Talk about what's new.

Fixes https://github.com/kubewarden/kubewarden.io/issues/136

### CODE REVIEW

**Review by jvanz — APPROVED** [review]

File: content/blog/2022/10/kubewarden_1_3_release.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+We are proud to announce that Kubewarden is currently A-rated, with a score above
+the 90%. The results are completely transparent and can be found by visiting
+the [Kubewarden dashboard](https://clomonitor.io/projects/cncf/kubewarden)
+on the CLOMonitor website.
+
+## Performance improvements
+
+We have reduced the startup time of Policy Server, the component in charge of
+running all our WebAssembly policies.
+The difference is quite significant when multiple policies are enforced by the
```
Comment (viccuad) [inline content/blog/2022/10/kubewarden_1_3_release.md:33]: would it make sense to mention it was an order of magnitude faster?

File: content/blog/2022/10/kubewarden_1_3_release.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+same `PolicyServer` Kubernetes resource.
+
+To achieve that, we had to perform some refactoring of the internal architecture
+of Policy Server. The good news is that this work paves the way for a feature
+we will deliver shortly: being able to put a constraint on the evaluation
```
Comment (viccuad) [inline content/blog/2022/10/kubewarden_1_3_release.md:38]:
```suggestion
we will deliver shortly: being able to put a *custom* constraint on the evaluation
```
I'm just concerned that people may think policy evaluation didn't have timeouts (even if it's only the webhook ones)

File: content/blog/2022/10/kubewarden_1_3_release.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+time of each policy.
+
+Kubewardene policies can be developed using [Turing complete](https://en.wikipedia.org/wiki/Turing_completeness)
```
Comment (viccuad) [inline content/blog/2022/10/kubewarden_1_3_release.md:41]:
```suggestion
Kubewarden policies can be developed using [Turing complete](https://en.wikipedia.org/wiki/Turing_completeness)
```

File: content/blog/2022/10/kubewarden_1_3_release.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+[Sigstore](https://sigstore.dev) is a project that is changing the way software
+can be safely distributed.
+
+We've seen the potential of Sigstore and we've embraced the project not just to
+sign our own artifacts (binaries, container images and even policies), but also
+to offer policies capable of
+[verifying image signatures](https://artifacthub.io/packages/kubewarden/verify-image-signatures/verify-image-signatures).
+
+Starting from the 1.3.0 release, Kubewarden can handle Sigstore signatures
+produced using a PKCS11 tokens, like hardware security modules (HSM) and smart
```
Comment (viccuad) [inline content/blog/2022/10/kubewarden_1_3_release.md:65]:
```suggestion
produced using a PKCS11 token, like hardware security modules (HSM) and smart
```

File: content/blog/2022/10/kubewarden_1_3_release.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+We have recently created some new Kubewarden policies. All of them are
+already available on [ArtifactHub](https://artifacthub.io).
+
+So far we've published a [blog post](/blog/2022/10/env-var-secrets/)
+dedicated to the
+[environment variable secret scanner](https://artifacthub.io/packages/kubewarden/env-variable-secrets-scanner/env-variable-secrets-scanner)
+policy.
+In next weeks we will publish other posts dedicated to the
+[volumeMount](https://artifacthub.io/packages/kubewarden/volumemounts-policy/volumemounts)
```
Comment (viccuad) [inline content/blog/2022/10/kubewarden_1_3_release.md:95]:
```suggestion
[volumeMounts](https://artifacthub.io/packages/kubewarden/volumemounts-policy/volumemounts)
```

**Review by viccuad — APPROVED** [review]
LGTM! minor nitpicks

File: content/blog/2022/10/kubewarden_1_3_release.md (hunk 1 above)
Comment (flavio) [reply content/blog/2022/10/kubewarden_1_3_release.md:33]: I don't have the data around anymore :/

Do you feel something like:

> Starting a Policy Server with multiple policies defined is now an order of magnitude faster

would be better?

File: content/blog/2022/10/kubewarden_1_3_release.md (hunk 2 above)
Comment (flavio) [reply content/blog/2022/10/kubewarden_1_3_release.md:38]: Good point, I'll find a way to address that

Comment (flavio) [reply content/blog/2022/10/kubewarden_1_3_release.md]: I removed this section as discussed during the daily

**Comment by flavio** [comment]
Can you please take one final look?

**Review by viccuad — APPROVED** [review]

**Pull request merged**